src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/EnumGroupAndName.cs

```diff
@@ -28,7 +30,28 @@ public EnumGroupAndName(string group, string name)
             }
 
             Group = group;
-            Name = name;
+            _nameFunc = () => name;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the EnumGroupAndName structure.
+        /// </summary>
+        /// <param name="group">The group name.</param>
+        /// <param name="name">A function which returns the name. (Necessary for proper localization.)</param>
+        public EnumGroupAndName(string group, Func<string> name)
```

Comment:
Probably should `[Obsolete]` the old constructor and file a 2.0 bug about removing it.